docs/rules/symbol-description.md

```diff
@@ -0,0 +1,61 @@
+# Require Symbol Description (symbol-description)
+
+The `Symbol` function may have optional description:
+
+```js
+var foo = Symbol("foo");
+
+var someString = 'bar';
+var bar = Symbol(someString);
+```
+
+
+Using `description` promotes easier debugging: when a symbol is logged the description is used:
+
+```js
+var foo = Symbol("foo");
+
+> console.log(foo);
+// Symbol(foo)
+```
```

Comment:
The variable name and the description should probably be different, as well, so it's clear exactly which `foo` is being shown in the console.log statement.